lib/rollout.rb

```diff
@@ -1,28 +1,30 @@
 require "rollout/version"
 require "zlib"
 require "set"
+require "json"
 
 class Rollout
   class Feature
-    attr_accessor :groups, :users, :percentage
+    attr_accessor :groups, :users, :percentage, :data
     attr_reader :name, :options
 
     def initialize(name, string = nil, opts = {})
       @options = opts
       @name    = name
 
       if string
-        raw_percentage,raw_users,raw_groups = string.split("|")
+        raw_percentage,raw_users,raw_groups,raw_data = string.split("|")
         @percentage = raw_percentage.to_f
         @users = (raw_users || "").split(",").map(&:to_s).to_set
         @groups = (raw_groups || "").split(",").map(&:to_sym).to_set
+        @data = JSON.parse(raw_data || "{}", symbolize_names: true)
```

Comment:
if case `raw_data` is blank we should probably avoid calling `JSON.parse` to avoid useless parse for feature which doesn't even have any raw data

so maybe something like

```
@data = JSON.parse(raw_data, symbolize_names: true) unless raw_data.blank?
```

------

Also what's the motivation for using `symbolize_names`? Older version of ruby don't do garbage collection for symbols and I'm afraid this can potentially cause some memory issues.